### ISSUE TITLE
git HEAD - getting ValueError: Invalid isoformat string error in gps9_date_base

### ISSUE THREAD

**Issue by KyleGW** [time4tea/gopro-dashboard-overlay#166]
Fresh install on ubuntu 22.04 and clean pull from github of current code, getting below error. Not sure if actual problem with the code or i made a mistake somewhere with initial install/setup. 

![image](https://github.com/time4tea/gopro-dashboard-overlay/assets/9725098/822e46b7-2d9e-4451-ac1a-409fe856d6f1)

**Comment by time4tea** [comment]
I think maybe you made a mistake with setting up the code. The way to set things up for development is a bit different. I wouldn't recommend using the code directly from git unless you planned to make changes.

If you clone the code from git, you can't then also install the package, so the "venv/bin/gopro-dashboard.py" looks a bit odd.

I think using the code from git would look something like this:  

```shell
git clone ...
cd gopro-dashboard-overlay
python3.11 -mvenv venv
venv/bin/pip3 install -r requirements-dev.txt
PYTHONPATH=. bin/gopro-dashboard.py ....
```

Or use PyCharm...

**Comment by time4tea** [comment]
@KyleGW - did you have any success?

**Comment by time4tea** [comment]
Hmmm. I have compiled a copy of python3.10, and it does look like this line doesn't in fact work on python3.10. I am now using python3.11 as a base, but I do want it to work on 3.10, if possible. I don't know why this line doesn't work as it is a valid iso datetime string afaik.

**Comment by time4tea** [comment]
It looks like this very specific change was made to datetime.fromisofomat in 3.11 - previously it couldn't parse the "Z". I'll see if i can find a better way of running the tests against multiple versions of python, as up till now its been "best efforts",

**Comment by time4tea** [comment]
Should be fixed in 0.120.0 - now on pypi (or HEAD) - also updated build to check multiple python versions. Thanks for highlighting!

**Comment by KyleGW** [comment]
updated and was then able to run the program. had to fix a couple other small items related to this newly created os install, but all up and running no problem now. Thanks!
